docs/contrib_components/api/installation.rst

```diff
@@ -0,0 +1,29 @@
+Installing Wagtail API
+======================
+
+The ``wagtailapi`` module can be installed with ``pip``:
+
+.. code-block:: shell
+
+    pip install wagtailapi
+
+
+Once installed, you then need to add ``wagtailapi`` to ``INSTALLED_APPS`` in your Django settings and configure a URL for it in ``urls.py``
+
+.. code-block:: python
+
+    # settings.py
+
+    INSTALLED_APPS = [
+        ...
+        'wagtailapi',
```

Comment:
The import path is now `wagtail.contrib.wagtailapi`, this appears to be outdated.